app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+from app.errors import (
```

Comment:
PEP8 suggests adding a blank line here https://peps.python.org/pep-0008/#imports

Reply:
like this?
```
from app.errors import (

    NotVaccinatedError,
    OutdatedVaccineError,
    NotWearingMaskError
)
```

Reply:
no, like this
```python
import datetime

from app.errors import (
```